README.md

```diff
@@ -103,7 +105,9 @@ module.exports = {
     ]
   }
   htmlLoader: {
-  	ignoreCustomFragments: [/\{\{.*?}}/]
+  	ignoreCustomFragments: [/\{\{.*?}}/],
+  	root: path.resolve(__dirname, 'assets'),
+  	attrs: ['img:src', 'link:href']
```

Comment:
Could you correct the spacing in this section when touching it? It is mixed tabs and spaces. Should be only spaces here.